src/components/app/details/appConfig/AppConfig.tsx

```diff
@@ -167,6 +172,15 @@ function getNavItems(isUnlocked: AppStageUnlockedType, appId: string): { navItem
             flowCompletionPercent: completedPercent,
             currentStep: completedSteps,
         },
+        {
+            title: 'External Links',
+            href: `/app/${appId}/edit/external-links`,
```

Comment:
Can we use an enum for `external-links` and same for above

Reply:
As discussed, we'll cover this in the upcoming task.